src/download.py

```diff
@@ -39,12 +35,16 @@ class Config:
         n += 1
         print(f"Page {n}")
         response = requests.get(url, headers=headers)
+
+        if response.status_code != 200:
+            raise Exception(f"Error {response.status_code}: {response.text}")
+
         data = response.json()
         res0 += data["results"]
 
     filename = resource.split("?")[0]  # To get rid of "?questions"
     filename = f"{filename}_latest.json"
-    filepath = f"../data/raw/{Config.event}/{filename}"
+    filepath = Path.joinpath(Config.raw_path, filename)
```

Comment:
I think here (and in other places were we have Pathlib) we can just do ```
```suggestion
    filepath = Config.raw_path / filename
```